src/routes/dashboard/fragment-wizard/choose-fragment-modal.tsx

```diff
@@ -32,6 +35,47 @@ const fragmentOptions = css`
 		flex-direction: column;
 	}
 `;
+const tileWrapper = css`
```

Comment:
export these because they are similar to the ones from the dashboard tile components